Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -180,8 +181,8 @@ - (void)createViews {
     [_submitCodeSpinner autoSetDimension:ALDimensionWidth toSize:kSpinnerSize];
     [_submitCodeSpinner autoSetDimension:ALDimensionHeight toSize:kSpinnerSize];
     [_submitCodeSpinner autoVCenterInSuperview];
-    [_submitCodeSpinner autoPinEdgeToSuperviewEdge:ALEdgeRight withInset:kSpinnerSpacing];
-    
+    [_submitCodeSpinner autoPinTrailingToSuperViewWithMargin:kSpinnerSpacing];
```

Comment:
Fixed.  See above - you were right, the trailing methods needs to invert, not invert-only-if-RTL their constants.